docker/aarch64/CrossCompile.Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM ubuntu:22.04 as builder
```

Comment:
suggestion: Maybe the file name `CrossCompile.Dockerfile` is too long, just `Dockerfile` is ok.